content/docs/tools-ui/uidev-control-gallery.md

```diff
@@ -0,0 +1,25 @@
+---
+linktitle: Control Gallery Tool
+title: O3DE Qt Control Gallery Tool
+description: Learn about the Open 3D Engine (O3DE) Qt Control Gallery tool that demonstrates the O3DE UI custom Qt widget library in action, and see sample code related to each example.
+weight: 100
+toc: true
+---
+
+Use the **O3DE Qt Control Gallery** tool to see the O3DE UI custom Qt widget library in action. Interact with the controls, and peruse sample code and working examples of selected components.
+
+![O3DE Qt Control Gallery tool](/images/tools-ui/o3de-qt-control-gallery.png)
+
+## How to use the O3DE Qt control gallery
+
+Before you begin, make sure that you have built or installed O3DE on your computer.
+
+1. Open your computer's file explorer to the O3DE binaries directory. If you installed the O3DE engine, the path on your computer will be similar to `<engine>/bin/Windows/profile`. If you build O3DE from source, the tool will build to your engine build directory.
+
+1. Locate and then launch `O3DEQtControlGallery.exe`.
+
+    ![O3DE Qt Control Gallery file location](/images/tools-ui/o3de-qt-control-gallery-tool-location.png)
+
+1. Use the dropdown menu in the upper left of the tool to navigate through the available components. For each component, the right side of the tool provides an interactive example of how the widget looks and works. The left side contains sample code related to the component.
+
+**Note:** The gallery does not cover how to set up a window for your tool, or the basics of how the Qt library works. Its primary objective is to illustrate the styles and features of the components in the library.
```

Comment:
Should we use a Hugo {{ < note > }} shortcode instead?

Reply:
I was on the fence, but since you suggested it, sure why not! ;)